app/models/site_configuration.rb

```diff
@@ -22,7 +22,7 @@ class SiteConfiguration
 
   DEFAULT_LOGO_PATH = "sites/logo-default.png".freeze
 
-  MODULES_WITH_NOTIFICATONS = ["GobiertoPeople", "GobiertoBudgetConsultations"]
+  MODULES_WITH_NOTIFICATONS = ["GobiertoPeople", "GobiertoBudgetConsultations", "GobiertoParticipation"]
```

Comment:
Could you add a reference to this constant in https://github.com/PopulateTools/gobierto/blob/master/docs/developing-module.md ?